Class-14-(SQL-Part-1).md

```diff
@@ -0,0 +1,26 @@
+# Class 14: Databases and SQL
+
+## In the prep work this week, the students learned
+1. What a relational database is.
+1. What SQL is and why it is important.
+1. How to perform CRUD operations on a database.
+1. How joins work.
+
+## Announcements
+1. Students are over half way done with Unit 2. Give them a shout out!
+1. Graded Assignment #4 covers the material from Classes 14-17. As always encourage the students to take the time to read through the whole assignment and see what they can get started on now.
+1. Check with your course manager for any important announcements.
+
+## Large Group Time (Instructor)
+
+### Class 14 Topics That Require Careful Attention
+1. Review what a relational database is. Databases seem simple, but can be kind of weird. Go over how tables store one type of info and are connected to each other.
+1. Review the different CRUD operations and retiterate warnings for updating and deleting data.
+1. Joins can be conceptually odd. Take the time on each one to go over a query and have students discuss what the result set might look like before sharing the result set.
+1. Review the studio!
+1. Before students head out for studio time, let students know that if they had issues installing MySQL and/or MySQL Workbench, resolving those issues come first!
+
+## Small Group Time: Class 14 Studio (TA Notes)
```

Comment:
I'd add a caution that the students need to use the SECOND SQL command for setting up the ``movies`` table. The first example does not establish a foreign key.

Reply:
Done! Thanks!